lib/pages/chat/chat.dart

```diff
@@ -792,7 +745,7 @@ class ChatController extends State<Chat>
     inputFocus.requestFocus();
   }
 
-  void scrollToEventIdAndHighlight(String eventId) {
+  Future<void> scrollToEventIdAndHighlight(String eventId) {
```

Comment:
Why need to update `Future`

Reply:
so that it can be put inside TwakeDialog
<img width="603" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/43041967/b1f02c3f-fb47-4eda-aaf2-578579536e39">